data/moves/en.json

```diff
@@ -1773,7 +1773,7 @@
     "damage": 16,
     "duration": 1200,
     "energy": 8,
-    "dps": 13.33
+    "dps": 13.3
```

Comment:
If you update this move file, please also do the French and German one. Else, setups in those langs will break